app/lib/methods/loadSurroundingMessages.ts

```diff
@@ -45,18 +47,18 @@ export default function loadSurroundingMessages({ messageId, rid }) {
 							_id: generateLoadMoreId(lastMessage._id),
 							rid: lastMessage.rid,
 							tmid,
-							ts: moment(lastMessage.ts).add(1, 'millisecond'),
+							ts: moment(lastMessage.ts).add(1, 'millisecond').toDate(),
 							t: MESSAGE_TYPE_LOAD_NEXT_CHUNK,
 							msg: lastMessage.msg
-						};
+						} as IMessage;
```

Comment:
This part is looking really weird to me.
By what type, you are returning an IMessage, but the IMessage doesn't only have this data.
Another thing is the reassing of the message in several places, I think we could do the approach of using const and naming the constants correctly.

Reply:
Now it looks better.
Nice :)